src/clegoues/genprog4java/mut/edits/java/JavaEditFactory.java

```diff
@@ -241,7 +241,7 @@ public JavaEditOperation makeEdit(Mutation edit, Location dst, EditHole sources)
 				int atom = item.getAtom();
 				List<WeightedAtom> inScopeThere = this.scopeHelper(variant.instantiateLocation(atom, item.getRight()), variant, editType);
 				for (WeightedAtom there : inScopeThere) {
-					if (there.getAtom() != location.getId()) { 
+					if (there.getAtom() == location.getId()) {
```

Comment:
not sure who uses SWAP but it doesn't make sense to swap statements when the location of the third expr is *not* the same as the original location